### PR TITLE
cat-592 now driven from nomis for complete cat history

### PR DESCRIPTION
**Pull request by lbennett-moj**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -32,5 +33,7 @@ knex1.migrate.latest().then(() => {
     log.info(`Server listening on port ${app.get('port')}`)
   })
   // must be created after migration scripts are complete
-  sqsService.app.start()
+  if (config.sqs.enabled === 'true') {
+    sqsService.app.start()
+  }
```
Comment (lbennett-moj) [inline server.js]: defaults to true - can now use SQS_ENABLED = false in env to disable sqs consumer and avoid the log errors

File: integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/model/DatabaseUtils.groovy [hunk 2]
```diff
@@ -63,6 +63,10 @@ class DatabaseUtils {
     sql.executeUpdate("update form set nomis_sequence_no = $seq where booking_id = $bookingId")
   }
 
+  def createNomisSeqNoWhenMultipleCatgorisationsForOffender(bookingId, seq, nomisSeq) {
```
Comment (steverendell) [inline integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/model/DatabaseUtils.groovy]: typo ^ (pedant mode)

File: server.js (hunk 1 above)
Comment (steverendell) [reply server.js]: 👍

File: server/services/offendersService.js [hunk 3]
```diff
@@ -861,6 +861,33 @@ module.exports = function createOffendersService(nomisClientBuilder, formService
...
+      nomisRecords.map(async nomisRecord => {
+        const foundCatRecord = catRecords.find(
+          o => o.bookingId === nomisRecord.bookingId && o.nomisSeq === nomisRecord.assessmentSeq
+        )
+        return {
+          ...nomisRecord,
+          prisonDescription: await getOptionalAssessmentAgencyDescription(context, nomisRecord.assessmentAgencyId),
+          recordExists: !!foundCatRecord,
+          assessmentDateDisplay: dateConverter(nomisRecord.assessmentDate),
+          sequence: foundCatRecord ? foundCatRecord.sequence : undefined,
```
Comment (steverendell) [inline server/services/offendersService.js]: i think foundCatRecord && foundCatRecord.sequence will work the same

File: server/services/offendersService.js [hunk 4]
```diff
@@ -861,6 +861,33 @@ module.exports = function createOffendersService(nomisClientBuilder, formService
...
+        }
+      })
+    )
+
+    return {
+      details,
+      history: dataDecorated.sort((a, b) => sortByDateTime(a.assessmentDateDisplay, b.assessmentDateDisplay)),
```
Comment (steverendell) [inline server/services/offendersService.js]: possibly should be approval date? i think this is more significant

**Pull request merged**